vita3k/shader/src/translator/alu.cpp

```diff
@@ -468,11 +468,12 @@ spv::Id USSETranslatorVisitor::do_alu_op(Instruction &inst, const Imm4 dest_mask
         break;
     }
 
+    case Opcode::ISUBU16:
+    case Opcode::ISUB16:
     case Opcode::ISUBU32:
-    case Opcode::ISUB32: {
+    case Opcode::ISUB32:
         result = m_b.createBinOp(spv::OpISub, m_b.makeIntType(32), vsrc1, vsrc2);
```

Comment:
Is `m_b.makeIntType(32)` right for ISUBU16 and ISUB16? Shouldn't be  `m_b.makeIntType(16)`? 🤔

Reply:
Integer are all extracted as 32-bit. The storing of them is 16-bit

Reply:
Yes, kd explain after i try 16 and get error you can't using 16-bit, need 32, check value already here